wcomponents-test-lib/src/test/java/com/github/bordertech/wcomponents/test/components/WTextAreaUI.java

```diff
@@ -0,0 +1,49 @@
+package com.github.bordertech.wcomponents.test.components;
+
+import com.github.bordertech.wcomponents.ActionEvent;
+import com.github.bordertech.wcomponents.Message;
+import com.github.bordertech.wcomponents.WApplication;
+import com.github.bordertech.wcomponents.WButton;
+import com.github.bordertech.wcomponents.WComponent;
+import com.github.bordertech.wcomponents.WFieldLayout;
+import com.github.bordertech.wcomponents.WMessages;
+import com.github.bordertech.wcomponents.WPanel;
+import com.github.bordertech.wcomponents.WTextArea;
+import com.github.bordertech.wcomponents.validation.ValidatingAction;
+
+import static com.github.bordertech.wcomponents.WFieldLayout.LAYOUT_FLAT;
+
+public class WTextAreaUI extends WApplication {
+
+	private WMessages messages = new WMessages();
+	private WPanel panel = new WPanel();
+
+	public WTextAreaUI() {
+
+		panel.add(messages);
+		WFieldLayout fieldLayout = new WFieldLayout(LAYOUT_FLAT);
+		fieldLayout.setTitle("this is a bunch of fields");
+
+		//WTextField
+		WTextArea wTextField = new WTextArea();
```

Comment:
Copy-paste and you haven't changed the member name. Is it wise to have a WTextArea called `wTextField`?